src/main/java/org/broadinstitute/hellbender/tools/spark/utils/IntHistogram.java

```diff
@@ -198,6 +202,46 @@ public boolean isDifferentByKSStatistic( final IntHistogram sampleHistogram, fin
             return false;
         }
 
+        public KSData evalKS( final IntHistogram sampleHistogram ) {
```

Comment:
It looks like this method is also only used in the `FindBreakpointEvidenceSpark` path that emits a lot more info about the data. It might be nice to comment on that and say when you might want to use this method as opposed to `isDifferentByKSStatistic`.